extensions/wikia/TemplateClassification/scripts/TemplateClassificationInCategory.js

```diff
@@ -82,6 +82,7 @@ define('TemplateClassificationInCategory',
 
 				mw.messages.set(res.messages);
 
+				// TODO fix while working on MAIN-10625
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)